cc/config/BUILD

```diff
@@ -42,11 +42,16 @@ config_setting(
 )
 
 config_setting(
-    name = "enable_gpu",
-    define_values = {"gpu": "1"},
+    name = "enable_tpu",
+    define_values = {"tpu": "1"},
 )
 
 config_setting(
-    name = "enable_tpu",
-    define_values = {"tpu": "1"},
+    name = "bazel_tf",
+    define_values = {"bazel_tf": "1"},
+)
+
+config_setting(
+    name = "never",
```

Comment:
I don't see this referenced anywhere, what's it for?

Reply:
I used this initially for checking a valid configuration, like this:

select({
  if_something(
    if_true = {"//conditions:default": [...]}
    if_false = {"//cc/config:never": []}
  ),
  no_match_error = "if_something is false",
})

No longer needed, removed.